manifest.php

```diff
@@ -31,7 +31,7 @@
     'label' => 'Test core extension',
     'description' => 'TAO Tests extension contains the abstraction of the test-runners, but requires an implementation in order to be able to run tests',
     'license' => 'GPL-2.0',
-    'version' => '14.3.0',
+    'version' => '14.4.0',
     'author' => 'Open Assessment Technologies, CRP Henri Tudor',
     'requires' => [
         'generis' => '>=12.15.0',
```

Comment:
i saw the update , great job , we still need something to be finished :
https://github.com/oat-sa/generis/pull/832
updates the version to  : 'version' => '13.3.0',
let's put that value here also :) 
thank you

Reply:
Thanks Mike! I've done this as well.